### PR TITLE
update build.gradle.kts for publishing

### PR DESCRIPTION
**Pull request by aegershman**
see: https://github.com/openrewrite/rewrite/issues/324

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -8,17 +8,17 @@
         <processorPath useClasspath="false">
```
Comment (aegershman) [inline .idea/compiler.xml:8]: (will remove this, just temporarily adding it as a draft to show itself is updating)

File: build.gradle.kts [hunk 2]
```diff
@@ -1,36 +1,45 @@
...
-import io.spring.gradle.bintray.SpringBintrayExtension
+//import io.spring.gradle.bintray.SpringBintrayExtension
 import nebula.plugin.contacts.Contact
 import nebula.plugin.contacts.ContactsExtension
 import nebula.plugin.info.InfoBrokerPlugin
 import nl.javadude.gradle.plugins.license.LicenseExtension
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
-import org.jfrog.gradle.plugin.artifactory.dsl.ArtifactoryPluginConvention
-import org.jfrog.gradle.plugin.artifactory.dsl.PublisherConfig
+//import org.jfrog.gradle.plugin.artifactory.dsl.ArtifactoryPluginConvention
```
Comment (jkschneider) [inline build.gradle.kts:8]: Could we just go ahead and remove these?

**Review by jkschneider — CHANGES_REQUESTED** [review]
It appears we have two different license plugins applied after this change. Can we just pick one?

File: build.gradle.kts (hunk 2 above)
Comment (aegershman) [reply build.gradle.kts:8]: yes definitely, I'm just committing this as I work through some of the compilation errors in subprojects

**Review by jkschneider — APPROVED** [review]

**Comment by aegershman** [comment]
* What went wrong:
Execution failed for task ':rewrite-core:publishNebulaPublicationToMavenLocal'.
> Failed to publish publication 'nebula' to repository 'mavenLocal'
   > Invalid publication 'nebula': multiple artifacts with the identical extension and classifier ('jar', 'null').


working on it

**Comment by aegershman** [comment]
closing this for the time being, will focus on other repos first before circling back

**Pull request closed without merging**